zrp/modeling/pipeline_builder.py

```diff
@@ -185,7 +187,7 @@ class ZRP_DataSampling(BaseZRP):
         Indicates the source of zrp_modeling data to use. There are three optional sources 'block_group', 'census_tract', and 'zip_code'. By default 'census_tract' is inferred.
     """
 
-    def __init__(self, zrp_model_source, file_path=None, zrp_model_name='zrp_0', *args, **kwargs):
+    def __init__(self, zrp_model_source, file_path=None, zrp_model_name='zrp_0', population_weights_dict=None, *args, **kwargs):
```

Comment:
`population_weights_dict`  is missing from the doc string, please add a description including details on how someone should add custom population weights

Reply:
done